rmgpy/data/thermo.py

```diff
@@ -1456,7 +1517,7 @@ def __addPolycyclicCorrectionThermoData(self, thermoData, molecule, polyring):
         # polycylic ring in molecule and match group)
         # otherwise, apply heuristic algorithm
         if not isPolyringPartialMatched(polyring, matched_group):
-            thermoData = addThermoData(thermoData, matched_group_thermodata, groupAdditivity=True)
+            thermoData = addThermoData(thermoData, matched_group_thermodata, groupAdditivity=True, verbose=True)
```

Comment:
Do we want verbose to be True or False here? I was under the impression that verbose is more for debugging purposes.